package-lock.json

```diff
@@ -34,7 +34,6 @@
         "@supy-io/ngx-intercom": "^14.2.12",
         "@swimlane/ngx-charts": "^20.1.2",
```

Comment:
@kivuvarosekivuvan for the `package-lock.json` file, let's approach it like this to resolve conflicts. And also because the package-lock.json shouldn't even be showing up here since it's generated (maintainers will do that work of generating it for contributors).

Restore the `package-lock.json` to original, the add and commit only the package-lock file, then push.

If your package-lock file changes again, just ensure **_not to_** add, commit and push it along with the other the files you have worked on.